avivator/src/components/Snackbars/SnackbarAlerts.jsx

```diff
@@ -6,14 +6,14 @@ export function OffsetsWarning() {
   return (
     <>
       A lot of channels have been detected in the requested OME-TIFF. To learn
-      how to speed up load times by providing byte offsets, refer to the first
-      point{' '}
+      how to speed up load times by providing byte offsets, refer to the point
+      about &quot;Indexed TIFF&quot;{' '}
```

Comment:
Think this can be much more brief. Should just link out to a more detailed summary, no need to go into detail.

```suggestion
     Avivator could not find an IFD index for the requested OME-TIFF, and therefore longer latencies are expected.
     Please read our documentation on &quot;Indexed TIFF&quot; to generate an IFD index for your OME-TIFF and improve initial load times.
```